src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -64,6 +65,8 @@ medToolBox::medToolBox(QWidget *parent) : QWidget(parent), d(new medToolBoxPriva
     connect(d->body, SIGNAL(maximized()), this, SLOT(behaveWithBodyVisibility()));
 
     this->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Minimum);
+
+    connect(this, SIGNAL(displayError(int)), this, SLOT(handleDisplayError(int)));
```

Comment:
OK, I think I see, you plan to do it inside the tbxes, right?
Thus I would do : 
`connect(job, SIGNAL(displayError(int)), this, SLOT(handleDisplayError(int)));`
inside the tbxes